editor/src/messages/input_mapper/input_mapper_message.rs

```diff
@@ -20,9 +20,15 @@ pub enum InputMapperMessage {
 	#[remain::unsorted]
 	#[child]
 	KeyUpNoRepeat(Key),
+	/// The only valid [Key]s for `DoubleClick` are:
+	/// - [Key::Lmb]
+	/// - [Key::Rmb]
+	/// - [Key::Mmb]
+	#[remain::unsorted]
+	#[child]
+	DoubleClick(Key),
```

Comment:
Try to avoid typing by comments. Either support double clicking any key or use `MouseKeys`.

Reply:
I pushed a halfway done commit trying to solve this but got stuck and really need to sleep now. Feel free to help finish it if you'd like. G'night sir!

Reply:
Since I couldn't figure out how to make this happen on my current attempt, I'm just going to leave a TODO comment for it and merge this so the PR doesn't get stale.

Reply:
I'll fix it @Keavon.

Reply:
Ok, thanks.